scripts/paper/aggregate_stwo_shared_table_reuse_benchmark.py

```
@@ -0,0 +1,126 @@
+    return (str(row["primitive"]), str(row["backend_variant"]), int(row["steps"]))
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--inputs", nargs="+", required=True, type=Path)
+    parser.add_argument("--output-json", required=True, type=Path)
+    parser.add_argument("--output-tsv", required=True, type=Path)
+    args = parser.parse_args()
+
+    if len(args.inputs) < 3:
+        raise SystemExit("expected at least 3 repeated benchmark runs for aggregation")
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)